### PR TITLE
feat: update multiple files in e2e node with framework helpers

### PR DESCRIPTION
**Pull request by draveness**
/kind cleanup
/priority backlog
/sig testing
/sig node 

**What this PR does / why we need it**:

Update multiple files in e2e node with framework helpers

**Which issue(s) this PR fixes**:

ref: #79686

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by draveness** [comment]
cc/ @oomichi

File: test/e2e_node/eviction_test.go
Comment (k-toyoda-pi) [inline test/e2e_node/eviction_test.go]: unnecessary line?

File: test/e2e_node/gpu_device_plugin_test.go
Comment (k-toyoda-pi) [inline test/e2e_node/gpu_device_plugin_test.go]: I think that `gomega.Expect...` at line:120 is unnecesarry.

File: test/e2e_node/eviction_test.go
Comment (draveness) [reply test/e2e_node/eviction_test.go]: done

File: test/e2e_node/gpu_device_plugin_test.go
Comment (draveness) [reply test/e2e_node/gpu_device_plugin_test.go]: done

**Comment by draveness** [comment]
/retest

Regards,
Draven

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm

Thanks for this clean up :)

**Comment by mattjmcnaughton** [comment]
/test pull-kubernetes-integration

I think this is flake, but if its a true failure, please ping me after fixing and I'll mark "lgtm" after the fix is merged :)

**Comment by draveness** [comment]
> /test pull-kubernetes-integration
> 
> I think this is flake, but if its a true failure, please ping me after fixing and I'll mark "lgtm" after the fix is merged :)

It's a flake, and thanks for your review

**Comment by oomichi** [comment]
Thanks

/approve

**Pull request merged**
